httpserver/src/test/kotlin/tbdex/sdk/httpserver/handlers/SubmitCloseTest.kt

```diff
@@ -0,0 +1,90 @@
+package tbdex.sdk.httpserver.handlers
+
+import ServerTest
+import TestData.aliceDid
+import TestData.createClose
+import TestData.createRfq
+import de.fxlae.typeid.TypeId
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+import tbdex.sdk.httpclient.models.ErrorResponse
+import tbdex.sdk.protocol.models.MessageKind
+import tbdex.sdk.protocol.serialization.Json
+import kotlin.test.assertContains
+import kotlin.test.assertEquals
+
+class SubmitCloseTest : ServerTest() {
+  @Test
+  fun `returns 400 if no request body is provided`() = runBlocking {
```

Comment:
nit: here and in other tests, the name describes the numeric value of the status code that should be returned (e.g. "400" in this test). Yet the implementation uses the actual name of the status code (e.g. `HttpStatusCode.BadRequest` in this test). This makes it harder for readers to verify whether the test is checking for what it's expected to do (since readers need to either know the http status codes, or look them up).